cmd/samples/cron/main.go

```diff
@@ -1,66 +1,125 @@
 package main
 
 import (
+	"context"
 	"flag"
 	"time"
 
 	"github.com/pborman/uuid"
+	"go.temporal.io/temporal-proto/serviceerror"
+	"go.temporal.io/temporal-proto/workflowservice"
 	"go.temporal.io/temporal/client"
 	"go.temporal.io/temporal/worker"
-
-	"github.com/temporalio/temporal-go-samples/cmd/samples/common"
```

Comment:
This is self contained sample w/o any dependencies. @mfateev, if you are ok with this approach, I will refactor all the rest.